Signum.Test/Environment/Entities.cs

```diff
@@ -129,7 +129,7 @@ public static class ArtistOperation
     }
 
     [Flags]
-    public enum Sex : short

```

Comment:
Why you removed this? This tries to test that enums work even for base types that are not int

Reply:
With the last extension/framework  changes the SQL migrations synchroniser throws errors with Int16 Enums. So we remove the short and then it works againafter conversion.